modules/thunder_liveblog/thunder_liveblog.info.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Thunder Liveblog'
```

Comment:
Or I could add an entity reference to liveblog at the end of an article. The downside of this could be that you can't easily mark your liveblogs in a special way on the frontpage, or make a view with all articles which have a liveblog.